nlp_architect/models/transformers/base_model.py

```diff
@@ -281,6 +281,8 @@ def _train(
         max_grad_norm: float = 1.0,
         logging_steps: int = 50,
         save_steps: int = 100,
+        training_args: str = None,
+        best_result_file: str = None,
```

Comment:
should not be here
if you want to save the best model this should be configured when building the model